### DISCUSSION TITLE
Annotating large files

### DISCUSSION THREAD

**Discussion by macdatar** [SkalskiP/make-sense#363] [Q&A]
Hello all, 

I just recently found out about makesense.ai and with it, I am able to annotate images with polygons and export coco-json file. However, what I want to do next is annotate orthmosaic 'tif' files that are really large in size (like 1gb and one of them is even 3gb). When I use the web interface, it does not upload the files so I installed it locally with npm and launched it in the browser locally. Still, when I specify a large file here, it doesnt seem to load and it just is in work mode for a while. 

![image](https://github.com/SkalskiP/make-sense/assets/426509/05137057-b442-4003-a808-c31e07b29bbf)

Any thoughts on how I could get around this issue?
Much appreciated.
regards,
Makarand

**Comment by macdatar** [comment]
I have an update (that happened fast). I was trying it with Chrome browser but when I use Safari, the file seems to be loading. Any idea what might be the case with Chrome?

**Comment by SkalskiP** [comment]
Hi @macdatar 👋🏻 I'm actually shocked that it worked at all. Make Sense does not support loading files in the `.tif` or `.tiff` formats.

**Reply by macdatar** [reply]
Hello @SkalskiP 
lol ok. Well, they are loading for me in Safari, can you try at your end? Also, when I installed node js, the forum said to install version 16.x.x but that didnt work for me (mac intel) so I installed node js 20.x.x and that came with npm 10.x.x and this worked for me. 
Here are some examples of tif files loaded in Safari, you can see the file name in the bottom. Can anything be done to increase the zoom level? Anything I can change in the source before npm start? (more text below the images)

![image](https://github.com/SkalskiP/make-sense/assets/426509/a0706594-fb2d-43d8-b1d6-0161f238912f)
![image](https://github.com/SkalskiP/make-sense/assets/426509/93a88658-885b-418a-962c-2d0580f98caf)
![image](https://github.com/SkalskiP/make-sense/assets/426509/16cc6ae6-8fdf-4c8a-b0b0-b6e12275358a)

Could you recommend anything else that could be used to annotate tif files in coco json? This seems to work for me but maybe something fishy?

Thank you so much for your help!

**Reply by SkalskiP** [reply]
I responded to zoom question [here](https://github.com/SkalskiP/make-sense/discussions/349#discussioncomment-9302013). :)

**Reply by macdatar** [reply]
Thanks!
